pkg/internal/mock/httpclient.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package mock contains common mocks of to be used on tests.
```

Comment:
`common mocks of` what? I think you either missed something or didn't mean to leave `of` in there